run.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+DB_FILE=osCam/db.sqlite3
+EMAIL_FILE=.env
+SETUP="GOOD"
+
+# Function to prompt user to enter email credentials
+setup_email(){
+
+    SETUP="GOOD"
+    if [[ ! -f "$EMAIL_FILE" ]]; then
+      echo "Would you like to configure your email settings before starting the app?"
+      read -p '[Y/N]: ' CONTINUE
+    else
+        break
+    fi
+
+    # If the user wants to configure collect their information and format the env file
+    if [[ "$CONTINUE" = "Y" ]]; then
+      read -p "Email Address: " EMAIL
+      read -p "Secret key: " SECRET_KEY
+
+      printf "EMAIL_HOST=smtp.gmail.com\n" > .env
+      printf "EMAIL_HOST_USER=$EMAIL\n" >> .env
+      printf "EMAIL_HOST_PASSWORD=$SECRET_KEY\n" >> .env
+      printf "RECIPIENT_ADDRESS=$EMAIL\n" >> .env
+
+    # Else just continue booting the server as normal
+    elif [[ "$CONTINUE" = "N" ]]; then
+      break
+    else
+      echo "Invalid character. Exiting"
+      SETUP="INVALID"
```

Comment:
if SETUP is invalid, should we loop until valid or user decides not to use this setting?